src/test/java/com/wikia/webdriver/testcases/auth/LoginTests.java

```diff
@@ -1,74 +1,253 @@
 
-  @Test(groups = "Login_anonCanLoginOnAuthModalFromGlobalNavigation")
-  public void anonCanLoginOnAuthModalFromGlobalNavigation() {
-    WikiBasePageObject base = new WikiBasePageObject();
-    base.openWikiPage(wikiURL);
-    NavigationBar signInLink = new NavigationBar(driver);
-    DetachedSignInPage authModal = new DetachedSignInPage(signInLink.clickOnSignIn());
+  @Test(groups = MOBILE)
+  @InBrowser(browser = Browser.CHROME, emulator = Emulator.GOOGLE_NEXUS_5)
+  public void userCanLogInAsStaffOnMobile() {
+    ArticlePage article = openArticleOnMobile();
+    loginOnMobileAs(article, staff);
+    article.waitForPageReload().verifyUserLoggedIn(staff);
+  }
 
-    authModal.login(credentials.userName10, credentials.password10);
-    base.verifyUserLoggedIn(credentials.userName10);
+  @Test(groups = DESKTOP)
+  public void japaneseUserCanLogInOnDesktop() {
+    ArticlePageObject article = openArticleOnDesktop();
+    loginOnDesktopAs(japaneseUser);
+    article.verifyUserLoggedIn(japaneseUser);
   }
 
+  @Test(groups = MOBILE)
+  @InBrowser(browser = Browser.CHROME, emulator = Emulator.GOOGLE_NEXUS_5)
+  public void japaneseUserCanLogInOnMobile() {
+    ArticlePage article = openArticleOnMobile();
+    loginOnMobileAs(article, japaneseUser);
+    article.waitForPageReload().verifyUserLoggedIn(japaneseUser);
+  }
 
-  @Test(groups = "Login_anonCanLoginAsStaffOnAuthModalFromGlobalNavigation")
-  public void anonCanLoginAsStaffOnAuthModalFromGlobalNavigation() {
-    WikiBasePageObject base = new WikiBasePageObject();
-    NavigationBar signInLink = new NavigationBar(driver);
-    base.openWikiPage(wikiURL);
+  @Test(groups = DESKTOP)
+  @Execute(onWikia = MercuryWikis.DISCUSSIONS_5)
+  public void userIsRedirectedToDiscussionPageUponLogInFromDiscussionPageOnDesktop() {
+    PostsListPage discussionPage = new PostsListPage().open();
+    loginOnDesktopFromDiscussionPageAs(user);
+    assertTrue(discussionPage.waitForPageReload().isStringInURL(PostsListPage.FULL_PATH),
+      "User should be redirected to discussion post list view upon log in");
+  }
 
-    DetachedSignInPage authModal = new DetachedSignInPage(signInLink.clickOnSignIn());
+  @Test(groups = MOBILE)
+  @Execute(onWikia = MercuryWikis.DISCUSSIONS_5)
+  @InBrowser(browser = Browser.CHROME, emulator = Emulator.GOOGLE_NEXUS_5)
+  public void userIsRedirectedToDiscussionPageUponLogInFromDiscussionPageOnMobile() {
+    PostsListPage discussionPage = new PostsListPage().open();
+    loginOnDiscussionMobilePageAs(discussionPage, user);
+    assertTrue(discussionPage.waitForPageReload().isStringInURL(PostsListPage.FULL_PATH),
+      "User should be redirected to discussion post list view upon log in");
+  }
+
+  @Test(groups = DESKTOP)
+  public void passwordTogglerChangesPasswordVisibilityOnDesktop() {
+    SignInPage signInPage = openLoginModalOnDesktop();
+    signInPage.typePassword(user.getPassword());
 
-    //we are using userNameStaff2 because of PLATFORM-2502 and PLATFORM-2508
```

Comment:
Is it okay to remove this?

Reply:
yep! I checked the tickets mentioned, the issue is no longer there. Also we're using a dedicated `User.LOGIN_STAFF` now!